setup.ps1

```diff
@@ -28,15 +29,22 @@ Disable-ScheduledTasks
 Disable-IPv6To4
 if ($manual_install) {
     Disable-InternetExplorerESC
-    Edit-VisualEffectsRegistry
 }
+Edit-VisualEffectsRegistry
 Add-DisconnectShortcut
 
-Install-Chocolatey
-Install-VPN
-Join-Network $network
-Install-NSSM
+if ($network) {
+    Install-Chocolatey
+    Install-VPN
+    Join-Network $network
+}
 
-Install-NvidiaDriver $manual_install
-Set-ScheduleWorkflow $admin_username $admin_password $manual_install
+Disable-Devices
```

Comment:
Does the gpu extension install before custom script? I had to split up the script because driver installation required a restart. It's really nice to not have to do that :)

Reply:
Yes, always !

The trick is in the deployment template, [line 248](https://github.com/ecalder6/azure-gaming/pull/41/files#diff-31daff04ca8ce7fc1169448437a3334bR248) : The script deployment `DependsOn` the Nvidia extension, so it won't start before the Nvidia extension's deployment is finished. The Nvidia ext reboots during the process, so we can get rid of the reboot thanks to them already doing it for us :) . 

It is indeed really nice not to have to do that, and also helps a lot during testing !